internal/verifierapi/fees.go

```diff
@@ -81,3 +83,65 @@ func (v VerifierApi) GetPluginPolicyFees(policyId uuid.UUID) (FeeHistoryDto, err
 
 	return feeHistory.Data, nil
 }
+
+// TODO add auth
+func (v VerifierApi) GetPublicKeysFees(ecdsaPublicKey string) (FeeHistoryDto, error) {
+	url := fmt.Sprintf("/fees/publickey/%s", ecdsaPublicKey)
+	v.logger.Debug("Getting public key fees for public key: ", ecdsaPublicKey)
+	v.logger.Debug("URL: ", url)
+	response, err := v.get(url)
+	if err != nil {
+		v.logger.WithError(err).Error("failed to get public key fees")
```

Comment:
If you already return a wrapped error , don't log it here , whoever call `GetPublicKeysFees` to log it 

otherwise you double log it